code/chart.js

```diff
@@ -1,4 +1,34 @@
+//Font-family for the chart
+Chart.defaults.font.family = 'Source Code Pro, monospace';
```

Comment:
Love that you looked into and added the charts font - it makes the whole site very cohesive.